### DISCUSSION TITLE
Sentry RN SDK Crashes on Android devices with recent Play Services update. No communication from Sentry's side.

### DISCUSSION THREAD

**Discussion by romeobalta** [getsentry/sentry-react-native#3294] [General]
As seen on [google issue tracker](https://issuetracker.google.com/issues/300840851) Sentry RN SDK causes crashes on Android devices with recent Google Play Services updates. Multiple people have reported that the crashes stop after they remove Sentry from their app. Is anyone at Sentry currently investigating this as we can definitely see you're aware of the issue:

<img width="1119" alt="Screenshot 2023-09-19 at 17 21 31" src="https://github.com/getsentry/sentry-react-native/assets/7095569/71e7fe2e-f34d-45cd-ae25-8fbe29d80f33">

**Comment by markushi** [comment]
Hi @romeobalta thanks for reaching out, yes we're looking into this already! On our end the most recent updates [are shared on this issue](https://github.com/getsentry/sentry-unity/issues/1444). Please follow along there and provide additional context if you're seeing this issue on your end as well.
